app/package.json

```diff
@@ -40,7 +40,8 @@
     "node-json-db": "0.9.1",
     "request": "2.85.0",
     "semver": "5.4.1",
-    "wurl": "2.5.0"
+    "wurl": "2.5.0",
+    "sortablejs": "*"
```

Comment:
@kanishk98 use the latest working version.

Reply:
Fixed it and pushed the changes.